apps/src/code-studio/pd/application/teacher1920/Section3TeachingBackground.jsx

```diff
@@ -72,9 +72,9 @@ export default class Section3TeachingBackground extends LabeledFormComponent {
         {this.props.data.school && this.props.data.school === '-1' && (
           <div style={styles.indented}>
             {this.inputFor('schoolName')}
-            {this.inputFor('schoolDistrictName')}
-            {this.inputFor('schoolAddress')}
-            {this.inputFor('schoolCity')}
+            {this.inputFor('schoolDistrictName', {required: false})}
+            {this.inputFor('schoolAddress', {required: false})}
+            {this.inputFor('schoolCity', {required: false})}
```

Comment:
Why are we styling these not-required? It looks like they are required if they are rendered at all, based on the condition above and the dynamic-require logic below.

Reply:
Oops, I'm wrong about this.  Never mind!